### PR TITLE
Update docker README

### PR DESCRIPTION
**Pull request by andrezafeu**
Update docker README

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -24,6 +26,14 @@ Running backend tests is easy:
 
 `docker-compose run --rm constituent_rails rspec`
 
+## Migrations
+
+`docker-compose run --rm constituent_rails` + migration command
```
Comment (e91096) [inline docker/README.md]: this works for me!
just curious: what's the different between `docker-compose run --rm constituent_rails` and `docker-compose exec constituent_rails`?

File: docker/README.md [hunk 2]
```diff
@@ -24,6 +26,14 @@ Running backend tests is easy:
...
+
+Example:
+
+`docker-compose run --rm constituent_rails rake db:migrate`
```
Comment (e91096) [inline docker/README.md]: can we use the whole `bundle exec rails db:migrate` as the migration command?

File: docker/README.md (hunk 1 above)
Comment (andrezafeu) [reply docker/README.md]: `exec` runs a command in an existing container, while `run` runs it on a new one

**Comment by e91096** [comment]
👍

**Pull request merged**